zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala

```diff
@@ -102,6 +103,16 @@ trait ZipkinWebFactory { self: App =>
 
 object Main extends TwitterServer with ZipkinWebFactory {
   def main() {
+    // If a scribe host is configured, send all traces to it, otherwise disable tracing
+    val scribeHost = sys.env.get("SCRIBE_HOST")
+    val scribePort = sys.env.get("SCRIBE_PORT")
+    if (scribeHost.isDefined || scribePort.isDefined) {
+      host.parse(scribeHost.getOrElse("localhost") + ":" + scribePort.getOrElse("1463"))
+      initialSampleRate.parse("1.0")
+    } else {
+      initialSampleRate.parse("0.0")
```

Comment:
Short answer is yes! I have a different impl which I will push when I get
connected later.